### PR TITLE
feat: logs change

### PR DESCRIPTION
**Pull request by sagarnaikjuspay**
## Type of Change

- [X] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

**Comment by koushikg14** [comment]
Let us remove systemic metrics tiles everywhere available on the dashboard as the new event viewer table has only the data beginning today.

**Comment by koushikg14** [comment]
would the name be changed from "Default Value" to value coming from api_flow ex: "api_flow": "PaymentsCreate"

image

File: .env
Comment (JeevaRamu0104) [inline .env:1]: Please don't change the env values

File: config/FeatureFlag.json
Comment (JeevaRamu0104) [inline config/FeatureFlag.json:1]: don't change the feature flag value and don't commit this file

**Review by JeevaRamu0104 — APPROVED** [review]

**Pull request merged**